### ISSUE TITLE
Use rafter-upload-service for creating (C)AssetGroup for testing testing-addon

### ISSUE THREAD

**Issue by magicmatatjahu** [kyma-project/addons#131]
**Description**

TL;DR;
Use [`rafter-upload-service`](https://github.com/kyma-project/rafter/tree/master/charts/rafter-upload-service) for creating AssetGroup in testing testing-addon in Kyma.

Before `1.9` Kyma release, we have a bug in release candidate branch with external brokers (aws, azure, gcp). We have wrong port pointed to `rafter-upload-service` inside deployment configuration of helm-broker, and provisioning of broker always failed, because data (files) required to create ClusterAssetGroup for some broker's ServiceClasses were sent to not existing port.
 
**Reasons**

For testing all functionalities of Rafter in documentation creation step.
